File: tests/samples/python/car.py
class Car:
    a = 5
    b = "hello"

    def __init__(self, brand, color, year):
        self.brand = brand
        self.color = color
        self.year = year

    def start_engine(self):
        print(f"The {self.color} {self.brand} engine has started.")

    def stop_engine(self):
        print(f"The {self.brand} engine has stopped.")


class Vehicle:
    def __init__(self, make, model):
        self.make = make
        self.model = model

File: tests/samples/swift/car.swift
import Foundation

class Car {
    var brand: String
    var color: String
    var year: Int
    
    init(brand: String, color: String, year: Int) {
        self.brand = brand
        self.color = color
        self.year = year
    }
    
    func startEngine() {
        print("The \(color) \(brand) engine has started.")
    }
    
    func stopEngine() {
        print("The \(brand) engine has stopped.")
    }
}

struct Vehicle {
    var make: String
    var model: String
}